internal/integritymonitor/integritymonitor.go

```diff
@@ -218,21 +220,19 @@ func integrityCheckFailed(
 	kubeClient *services.KubeClient,
 ) {
 	l := log.WithError(err)
-	var path string
+	var mPath string
 	var integrityError *IntegrityError
 	if errors.As(err, &integrityError) {
-		path = integrityError.Path
+		mPath = integrityError.Path
 		l = l.WithField("path", integrityError.Path)
 	}
 
 	l.Error("check integrity failed")
 
-	e := alerts.Send(alerts.Alert{
-		Time:    time.Now(),
-		Message: fmt.Sprintf("Restart deployment %v", deploymentData.NameDeployment),
-		Reason:  err.Error(),
-		Path:    path,
-	})
+	e := alerts.Send(alerts.New(fmt.Sprintf("Restart deployment %v", deploymentData.NameDeployment),
+		err.Error(),
+		mPath,
+	))
```

Comment:
And may be, make this alerter (Splunk?) also optional?

Reply:
This is common alerts' registry, thus only registered alerts will be sent.